scripts/globals/mobskills/perdition.lua

```diff
@@ -0,0 +1,20 @@
+---------------------------------------------
+-- Perdition
+-- Description: Instant K.O.
+---------------------------------------------
+require("scripts/globals/monstertpmoves")
+require("scripts/globals/status")
+require("scripts/globals/msg")
+---------------------------------------------
+
+function onMobSkillCheck(target,mob,skill)
+    return 0
+end
+
+function onMobWeaponSkill(target, mob, skill)
+
+    skill:setMsg(dsp.msg.basic.FALL_TO_GROUND)
```

Comment:
should this check for  `dsp.mod.DEATHRES`?

Reply:
Ive been hit with it 2 times in retail. Both times i was lvl 99 with full 119 gear and the mob was 68 and 74 and it instantly dropped me both times

Reply:
2 times is a) a pretty small sample and b) if you aren't wearing any death resist gear it can't proc and save you (there is no native source of it)

Reply:
honestly I didn't know if it should be checked here but your reasoning for not doesn't really answer that.

Reply:
I think we should assume yes until data says no:
https://github.com/DarkstarProject/darkstar/blob/8b920c7a9ad7db8a5f8e62dac24c32590728ff54/scripts/globals/spells/death.lua#L15

Reply:
Ill add it tonight. My retail account has expired now to do anymore testing